test/karma/firefox.js

```diff
@@ -0,0 +1,8 @@
+var common = require('./common');
+
+module.exports = function(config) {
+  config.set(common({
+    plugins: ['karma-firefox-launcher'],
+    browsers: ['Firefox']
+  }));
+};
```

Comment:
I think we have all these files so that you can test one specific browser at a time example: 
npm run test:chome. 

There is also a "detected" file which will detect which browsers you have available and test on all of them. This is the main thing that will run when you do npm run test.